hack/lib/images.bash

```diff
@@ -5,7 +5,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/metadata.bash"
 
 registry_host='registry.ci.openshift.org'
 registry="${registry_host}/openshift"
-export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"main"}
+export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"release-$(metadata.get project.version)"}
```

Comment:
:heart_eyes:

Reply:
Does it make sense to use just
```
export CURRENT_VERSION_IMAGES="release-$(metadata.get project.version)"
```
and then leave the file `.github/workflows/validate.yaml` untouched?